test/test_installation.py

```diff
@@ -181,7 +181,10 @@ def install_umf(self) -> None:
                     f"Error: Installation directory '{self.install_dir}' is not empty"
                 )
 
-        install_cmd = f"cmake --build {self.build_dir} --config {self.build_type.title()} --target install"
+        if platform.system() == "Windows":
+            install_cmd = f"cmake --install {self.build_dir} --config {self.build_type.title()} --prefix {self.install_dir}"
```

Comment:
hmm didn't we just circle back to the original issue: https://github.com/oneapi-src/unified-memory-framework/issues/971 ?

Perhaps we should have various CMakes tested also on Windows...?

Reply:
no, the #971 issue is related to Ubuntu. I added several tests in previous PRs to reproduce this case in our CI. But I'll add testing of various CMake on Windows